web/pdf_thumbnail_view.js

```diff
@@ -124,8 +124,8 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     this.anchor = anchor;
 
     var div = document.createElement('div');
-    div.id = 'thumbnailContainer' + id;
```

Comment:
There is code at https://github.com/mozilla/pdf.js/blob/e9c63a2b3274e7fc851fcdc0761bec7eefb3f4b6/web/pdf_thumbnail_viewer.js#L90 that relies on this being present, so that needs to be updated to use the `data-page-number` attribute as well.